### PR TITLE
/collectors/charts.d.plugin: remove deprecated modules

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

**This PR**:

-   removes deprecated bash modules (source code, mention in the docs)
-   adds `BREAKING_CHANGES.md`

___

During my work on https://github.com/netdata/netdata/issues/7878, i noticed that 90% of our bash modules are deprecated. I think it is time to do cleanup and remove them.

Fixing it in the installer can take some time, i think we can do it later. **The goal is to get rid of old unused code.**

Rel: #6151

Continuation of: #4104

Same method i used in https://github.com/netdata/netdata/pull/7853

##### Component Name

[`/collectors/charts.d.plugin`](https://github.com/netdata/netdata/tree/master/collectors/charts.d.plugin)

##### Additional Information

- [X] skip obsolete modules durin modules load in the charts.d.plugin (same way we do in in `python.d.plugin`)
- [x] remove obsolete modules
- [x] update documentation

removed modules:
- apache
- cpu_apps
- cpufreq
- exim
- hddtemp
- load_average
- mem_apps
- mysql
- nginx
- phpfpm
- postfix
- squid
- tomcat

___

### Tests:

- Clean install

No errors, result

```console
test@DEB10-1:~$ ls -l /opt/netdata/usr/libexec/netdata/charts.d/ /opt/netdata/usr/lib/netdata/conf.d/charts.d/
/opt/netdata/usr/libexec/netdata/charts.d/:
total 60
-rwxr-xr-x 1 root root  5244 Feb  3 16:46 ap.chart.sh
-rwxr-xr-x 1 root root  6644 Feb  3 16:46 apcupsd.chart.sh
-rwxr-xr-x 1 root root  3667 Feb  3 16:46 example.chart.sh
-rwxr-xr-x 1 root root  4860 Feb  3 16:46 libreswan.chart.sh
-rwxr-xr-x 1 root root  7124 Feb  3 16:46 nut.chart.sh
-rwxr-xr-x 1 root root 13285 Feb  3 16:46 opensips.chart.sh
-rwxr-xr-x 1 root root  7599 Feb  3 16:46 sensors.chart.sh

/opt/netdata/usr/lib/netdata/conf.d/charts.d/:
total 28
-rw-r--r-- 1 root root 652 Feb  3 16:46 ap.conf
-rw-r--r-- 1 root root 653 Feb  3 16:46 apcupsd.conf
-rw-r--r-- 1 root root 613 Feb  3 16:46 example.conf
-rw-r--r-- 1 root root 835 Feb  3 16:46 libreswan.conf
-rw-r--r-- 1 root root 882 Feb  3 16:46 nut.conf
-rw-r--r-- 1 root root 546 Feb  3 16:46 opensips.conf
-rw-r--r-- 1 root root 811 Feb  3 16:46 sensors.conf
```

- Update

`charts.d.plugin` skips obsolete modules

```console
root@DEB10-1:/opt/netdata/usr/libexec/netdata/plugins.d# ./charts.d.plugin debug 2>&1 | grep obsolete
2020-02-03 16:59:36: charts.d: DEBUG: apache: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: cpu_apps: is replaced by apps.plugin, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: cpufreq: is replaced by proc plugin, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: exim: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: hddtemp: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: load_average: is replaced by proc plugin, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: mem_apps: is replaced by proc plugin, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: mysql: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: nginx: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: phpfpm: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: postfix: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: squid: is replaced by python.d.plugin module, skipping it.
2020-02-03 16:59:36: charts.d: DEBUG: tomcat: is replaced by python.d.plugin module, skipping it.
```

### CODE REVIEW

**Comment by ilyam8** [comment]
i think we need to fix code owners for this bash stuff @cosmix 😄

File: collectors/charts.d.plugin/charts.d.plugin.in [hunk 1]
```diff
@@ -390,14 +390,34 @@ declare -A charts_enable_keyword=(
...
-	local charts= enabled= required=
+	local charts enabled required
 
 	# find all enabled charts
-
 	for chart in $(all_charts); do
 		MODULE_NAME="${chart}"
 
+		if [ -n "${obsolete_charts["$MODULE_NAME"]}" ]; then
+		  debug "is obsolete by ${obsolete_charts["$MODULE_NAME"]}, skipping it."
```
Comment (cakrit) [inline collectors/charts.d.plugin/charts.d.plugin.in]: "is replaced by" is better

Comment (cakrit) [reply collectors/charts.d.plugin/charts.d.plugin.in]: or maybe is obsoleted by

**Comment by ilyam8** [comment]
🎉  breaking changes 🎉

**Pull request merged**